PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -56,3 +57,8 @@
 
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 pp_on_AA_2018.toReplaceWith(slimmingTask, slimmingTask.copyAndExclude([slimmedOOTPhotons]))
+
+from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
+_phase2_timing_slimmingTask = cms.Task(slimmingTask.copy(),
+                                       offlineSlimmedPrimaryVertices4D)
+phase2_timing.toReplaceWith(slimmingTask,_phase2_timing_slimmingTask)
```

Comment:
@bendavid @slava77 given the other discussions in the review about phase2_timing vs phase2_timing_layer , is this really the desired behaviour?

Reply:
Yes, this adds an additional slimmed primary vertex collection to the miniaod.  In the case of phase2_timing alone, this corresponds to the fastsim 4d vertices, in the case of phase2_timing_layer, this corresponds to fullsim 4d vertices.